### PR TITLE
Add tabs and ecosystem tab content

### PR DESCRIPTION
**Pull request by Bluesmile82**
Add tabs in Industrial cases construction and food

Add content and structure for ecosystem. Also mobile

Update text with document

https://www.figma.com/design/R3bsPj03L8Rx3D36hHIBTa/NCP-Designs-%5BInternal%5D?node-id=3126-4811&t=Y75fwXfBtr00r7rm-4

### CODE REVIEW

File: src/app/industry-use-cases/page.tsx [hunk 1]
```diff
@@ -122,20 +130,120 @@ const TriggerContent = ({
...
     imageSrc2: "/assets/industry-cases-accordion-2.png",
+    content: <AccordionItemContent {...ACCORDION_ITEMS_CONTENT["constructions"]} />,
   },
   {
     id: "Tourism",
     text1: "Tourism depends on",
     text2: "Wetlands and Mangroves.",
     imageSrc1: "/assets/industry-cases-accordion-3.png",
     imageSrc2: "/assets/industry-cases-accordion-4.png",
+    content: <div>Byyeeee</div>,
```
Comment (clementprdhomme) [inline src/app/industry-use-cases/page.tsx]:
```suggestion
    content: <div>Coming soon!</div>,
```

File: src/components/industry-use-cases/accordion-item-content/index.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+              width={685}
+              height={388}
+            />
+            <div className="flex w-full -translate-y-[71px] justify-end text-white">
+              <div className="max-w-[724px] bg-black p-6 lg:p-10">{imageText}</div>
+            </div>
+          </div>
+          <div className="relative -mt-[71px] flex flex-col justify-between gap-10 text-pretty tracking-tight lg:flex-row lg:items-end lg:gap-4">
+            <div className="max-w-[395px]">{content2}</div>
+            <Image
```
Comment (clementprdhomme) [inline src/components/industry-use-cases/accordion-item-content/index.tsx:86]: Would be great if the image wasn't cut on the right side on the mobile layout.
The pebble image is cut on the right before reaching the edge of the screen

File: src/components/info-tooltip.tsx
Comment (clementprdhomme) [inline src/components/info-tooltip.tsx:1]: Could we have a better focus style on these buttons?

**Review by clementprdhomme — COMMENTED** [review]
Looks good!

File: src/app/industry-use-cases/page.tsx (hunk 1 above)
Comment (Bluesmile82) [reply src/app/industry-use-cases/page.tsx]: I updated the text and removed this for the moment

**Review by clementprdhomme — APPROVED** [review]

**Pull request merged**